### DISCUSSION TITLE
那个 大佬可以帮忙解决一下吗，4090双卡微调出现以下两个问题

### DISCUSSION THREAD

**Discussion by BXHQu** [zai-org/ChatGLM3#591] [Q&A] [answered]
### **1.loss第一个正常，第二个往后全为0**

{'loss': 2.7197, 'learning_rate': 0.0001999, 'epoch': 0.0}
{'loss': 0.0, 'learning_rate': 0.0001998, 'epoch': 0.0}
{'loss': 0.0, 'learning_rate': 0.00019970000000000003, 'epoch': 0.0}
{'loss': 0.0, 'learning_rate': 0.0001996, 'epoch': 0.0}
{'loss': 0.0, 'learning_rate': 0.00019950000000000002, 'epoch': 0.0}
{'loss': 0.0, 'learning_rate': 0.00019940000000000002, 'epoch': 0.0}
{'loss': 0.0, 'learning_rate': 0.00019930000000000002, 'epoch': 0.0}
{'loss': 0.0, 'learning_rate': 0.00019920000000000002, 'epoch': 0.0}
{'loss': 0.0, 'learning_rate': 0.00019910000000000001, 'epoch': 0.0}
{'loss': 0.0, 'learning_rate': 0.000199, 'epoch': 0.0}
{'loss': 0.0, 'learning_rate': 0.0001989, 'epoch': 0.0}
{'loss': 0.0, 'learning_rate': 0.0001988, 'epoch': 0.0}

**### 2.保持checkpoint时，出现文件不存在**
RuntimeError: Parent directory output/text-20231213-185108-2e-4/tmp-checkpoint-50 does not exist.
2%|▎ | 50/2000 [00:06<04:13, 7.68it/s]

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
上述问题有两种情况
1. 环境不对
2. 模型不是hf下的权重？
我解决了三个这种问题都是这两种
